experiments/dosarray_multi_experiment_dedos.sh

```diff
@@ -17,7 +17,7 @@ then
   echo "Need to configure DoSarray -- could not find dosarray_config.sh at \$DOSARRAY_SCRIPT_DIR/config (${DOSARRAY_SCRIPT_DIR}/config)" >&2
   exit 1
 fi
-source "${DOSARRAY_SCRIPT_DIR}/config/dosarray_config.sh"
```

Comment:
Was this line redundant?

Reply:
This line wasn't removed. I just added an additional space

Reply:
Agreed. I'll add it back